TestAssets/DesktopTestProjects/BindingRedirectSample/AppWithRedirectsAndConfig/AppWithRedirectsAndConfig.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
What did you use to migrate this project? I see a couple of things here that are no longer need: excludes and the definition the DefineConstants for NET46.